content/en/docs/tasks/tls/manual-rotation-of-ca-certificates.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Manual Rotation of CA Certificates
+min-kubernetes-server-version: v1.13
+content_template: templates/task
+---
+
+{{% capture overview %}}
+
+This page shows how to manually rotate the certificate authority (CA) certificates.
+
+{{% /capture %}}
+
+
+{{% capture prerequisites %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+For more information about authentication in Kubernetes, see [Authenticating](/docs/reference/access-authn-authz/authentication).
+For more information about best practices for CA certificates, see [Single root CA](docs/setup/best-practices/certificates/#single-root-ca).
+
+{{% /capture %}}
+
+{{% capture steps %}}
+## Choose an approach
+There are two ways to rotate or replace CA certificates.
+
+## Replace the cluster (recommended) {#replace-the-cluster}
```

Comment:
I would not recommend this approach.  Replacing a cluster to rotate trust isn't rotation, is destructive, causes service reference problems, and cause PVC issues.  I don't think we should provide this as an option.

Reply:
replacing the cluster is the safest approach, because one would be migrating the workload while doing so. At some point the approach should use a blue green kind of methodology to maintain operation.

Reply:
potentially if there are arguments against this approach it can be moved at the bottom as an "alternative" and undocumented option.

Reply:
On second thought @deads2k has mentioned a valid point. Although its not quite about destruction but about many other factors, cost is one of them. It is easy to move cluster when you have less workload and less number of nodes but when there is a large infrastructure, not everyone would agree for replacing the cluster as a recommended solution.

@neolit123  if you agree, can you please suggest how can we draft it as an alternative approach in this document.

Reply:
Hi @abhiTamrakar

You could have an “Alternatives” section at the end of the document, and a note at the top that tells the reader that this is quite a lot of work and to have a look there if they're not sure the approach is a good use of their time.

How does that sound?

Reply:
+1

Reply:
Added, it looks better now, waiting for approval, if there are no more changes :)